Server/Communication/router.py

```diff
@@ -49,9 +51,14 @@ def processRequest(message):
 	else:
 		data['session_username'] = user
 
+	# If the command is IL, get the most current version
```

Comment:
Do we have to do this part in the router class?  Can we do it in the intial load function?  I want to try and minimize the amount of logic we put in the main router class.